### ISSUE TITLE
[Electricraft] Motors uses wrong voltage and current

### ISSUE THREAD

**Issue by Holorizor** [ReikaKalseki/Reika_Mods_Issues#3074]
MC-Version: 1.7.10
Mods: DragonAPI, Rotarycraft V31b, Electricraft V31a, Reactorcraft V31b, NEI, chicken core
Mode: Singleplayer

Hello,

I have a simple Solartower battery motor set up from rotarycraft and electricraft. Solar tower power is converted into electrical power, stored in a battery, and from the battery, an electrical motor is powered.

The problem is, that the motor ignores the battery output and uses some other input, and changes its power output with the daytime.
Example: Batterie emits 4096 V and 256 A, but the motor (connected with only 1 isolated silver cable) produces only 372,008 kW at 728 Nm and 511 rad/s (so 91 A and 4095 V). The speed is correct because of the voltage loss over the silver cable, but the torque is not correct. With 256 A, I expect something like 2048 Nm. The Cable to the battery runs at this example (nighttime) with 4095 V and 19 A.

If I disconnect the battery from the solar tower, the motor produces the expected result. And at day, when the amperes on the battery charging cable increases the output of the motor on the battery output also increases.

Looks like a bug with the network and the batteries.

**Comment by Holorizor** [comment]
I did some more testing and I found out, that it is a problem with the motor and the upgrade status. I have only an inductive upgrade applied so 2 sources. My system has a generator from the solar tower, which charges two batteries, and one of them is used for the motor. In the network, this seems to be counted as 3 sources. A tungsten upgrade solved the problem. So the motor does count all batteries in the network and generators which are connected together in the network. 

This means I need to make generator-motor combinations to break down networks and overcome the 4 sources limit in bigger networks?
